src/index.ts

```diff
@@ -3,7 +3,12 @@
 // Licensed under the BSD-3-Clause license found in the LICENSE file or
 // at https://opensource.org/licenses/BSD-3-Clause
 
-export * from './group.js'
+import { GroupConsSjcl } from './groupSjcl'
+import { Oprf } from './oprf'
+
+Oprf.Group = GroupConsSjcl
+
+export * from './groupTypes.js'
```

Comment:
There should be a `Group` variable ready to use externally, without revealing details about its implementation. 


```suggestion
import { GroupConsSjcl } from './groupSjcl.js'
import { Oprf } from './oprf.js'

Oprf.Group = GroupConsSjcl

export { GroupConsSjcl as Group } from './groupSjcl.js'
export * from './groupTypes.js'
```

Reply:
Can someone not simply use Oprf.Group ? 
That's the way it's currently configured for the rest of the module ? 
I've updated the tests to simply assume Oprf.Group is configured (as it is now in index.js) and to use Oprf.Group for the tests.